### ISSUE TITLE
Suggestion of improvements from users

### ISSUE THREAD

**Issue by sblendorio** [xlar54/ultimateii-dos-lib#21]
User "Bela Erdos" suggests that:
https://www.facebook.com/groups/commodoresixtyfour/permalink/10155726173572172/?comment_id=10155729582457172

> Hey there, I sent an email too, but not sure which one will reach you quicker so I just post it here too.
> ok 1.61 works fine, 
> Some idea for feature update:
> 1 - let's name the BBS connection and browse by that name instead of the full link. 
> 2 - make a alphabetical sort option, right now as I filling up the list gets harder to find the one I'm looking for
> 3 - let's have a field for the named connection as Memo, so folks can add note to that bbs
> 4 - integrated password manager. Let's create a separate file where user can save his login credentials and after entering a master password can see it on top of the screen when they try to login. Obviously it is super important to make user aware of DO NOT SHARE YOUR PSSW file... :D for obvious reason, but would help to make the program better
> If anything else comes up, I'll let you know
> B3L4 / Hokuto Force

**Comment by xlar54** [comment]
Yes, agreed with these.  #4 we might want to think about how this would work.

#1 should be contained in the download from bbs outpost, so once we solve that issue, we should have the bbs name.  #2 can be sorted upon saving,  #3 should be easy enough.

We probably need to centralize the input/output code for windows, input boxes, etc.  At one point in time, i had worked on a turbo vision - style text UI system for the 128.  The input system for the phonebook was a rush job, but if we are going to get more intricate with inputs and such, we might want to consider improving that architecture.  Maybe youve already done some cleanup in those areas with your recent work. 

But i would break these into separate tickets, and we can pick them off as time allows.

**Comment by sblendorio** [comment]
I agree on making separate tickets for each suggestion, and on re-engineering window and input/output from user
